scanpipe/pipelines/scan_package.py

```diff
@@ -121,8 +118,8 @@ def build_inventory_from_scan(self):
         """
         project = self.project
         scanned_codebase = scancode.get_virtual_codebase(project, str(self.scan_output))
-        scancode.create_codebase_resources(project, scanned_codebase)
```

Comment:
@JonoYang should the `LoadInventory.build_inventory_from_scan` gets the same change?